Documentation/helm-operator.md

```diff
@@ -107,13 +107,16 @@ The following tables lists the configurable parameters of the rook-operator char
 | `agent.mountSecurityMode`    | Mount Security Mode for the agent.                                                                      | `Any`                                                  |
 | `agent.toleration`           | Toleration for the agent pods                                                                           | <none>                                                 |
 | `agent.tolerationKey`        | The specific key of the taint to tolerate                                                               | <none>                                                 |
+| `agent.nodeAffinity`         | The node labels for affinity of `rook-agent`                                                            | <none>                                                 |
 | `discover.toleration`        | Toleration for the discover pods                                                                        | <none>                                                 |
 | `discover.tolerationKey`     | The specific key of the taint to tolerate                                                               | <none>                                                 |
+| `discover.nodeAffinity`      | The node labels for affinity of `discover-agent`                                                        | <none>                                                 |
 | `mon.healthCheckInterval`    | The frequency for the operator to check the mon health                                                  | `45s`                                                  |
 | `mon.monOutTimeout`          | The time to wait before failing over an unhealthy mon                                                   | `600s`                                                 |
 
 &ast; For information on what to set `agent.flexVolumeDirPath` to, please refer to the [Rook flexvolume documentation](flexvolume.md)
 &ast; `agent.mounts` should have this format `mountname1=/host/path:/container/path,mountname2=/host/path2:/container/path2`
+&ast; `agent.nodeAffinity` and `discover.nodeAffinity` should have the format `"role=storage,rook; storage=ceph"` or `storage=;role=rook-example` or `storage=;` (_checks only for presence of key_)
```

Comment:
Could you make it three asteriks for the nodeAffinity point and add a second to the `agent.mounts` point to make the three points more distinguishable from each other.

Reply:
done.